### ISSUE TITLE
Enable inline javascript

### ISSUE THREAD

**Issue by pmkrawczyk** [nonoroazoro/docusaurus-plugin-less#1]
Hi, 

I'm trying to enable inline javascript in less by adding the following to docusaurus.config.js:

```
plugins: [
    ["docusaurus-plugin-sass", {}],
    [
      require.resolve("docusaurus-plugin-less"),
      {
        lessOptions: {
          javascriptEnabled: true,
        },
      },
    ],
  ],
```

But this does not work, inline javascript throws errors. 

By the way, while looking through the plugin code I found this:
```
const AVAILABLE_LOADER_OPTION_KEYS = [
    'lessOption',
    'additionalData',
    'sourceMap',
    'webpackImporter'
];
```
As far as I can see, the key should be 'lessOptions' (plural). Changing this does not solve my problem though...

Can anyone help, please?

**Comment by nonoroazoro** [comment]
Can you paste your code (including less) here?

**Comment by pmkrawczyk** [comment]
Thank you for your help!

The less file comes from the [RSuite](https://github.com/rsuite/rsuite), I am trying to import the _steps_ component into docusaurus.

The component imports properly but when I try to import styles using this command:
`import "rsuite/lib/Steps/styles";`

I get the following error: 

```
.tinyColorMixin();
^
Inline JavaScript is not enabled. Is it set in your options?
      in node_modules\rsuite\lib\styles\mixins\tiny-color.less (line 1183, column 0)
```
The error is thrown by this command:
`.tinyColorMixin();`
in [this _less_ file](https://github.com/rsuite/rsuite/blob/next/src/styles/mixins/tiny-color.less).

Is this sufficient information (I'm quite a noob at this!)

**Comment by nonoroazoro** [comment]
@pmkrawczyk Please use the following code to set less-loader options:

```
plugins: [
    [
      "docusaurus-plugin-less",
      { lessOptions: { javascriptEnabled: true } }
    ]
  ],
```

**Comment by pmkrawczyk** [comment]
Yes, I've tried this syntax as well, I'm afraid ti doesn't work either - I always get the mentioned errors...

**Comment by nonoroazoro** [comment]
It works in my project, weird...

**Comment by yijinc** [comment]
i have the same error 😥
